packages/flutter/test/material/input_chip_test.dart

```diff
@@ -137,20 +137,23 @@ void main() {
     final FocusNode focusNode2 = FocusNode(debugLabel: 'InputChip 2');
     await tester.pumpWidget(
       wrapForChip(
-        child: Column(
-          children: <Widget>[
-            InputChip(
-              focusNode: focusNode1,
-              autofocus: true,
-              label: const Text('Chip A'),
-              onPressed: () { },
-            ),
-            InputChip(
-              focusNode: focusNode2,
-              autofocus: true,
-              label: const Text('Chip B'),
-            ),
-          ],
+        child: Focus(
+          focusNode: FocusScopeNode(),
```

Comment:
Why did this need the extra focus scope node, and why not use a FocusScope widget?  Will apps need to do similar modifications after this change?

Reply:
`FocusScope` is much cleaner, thanks!

The reason I need it here (and also in `raw_material_button_test.dart` and in `text_field_test.dart`) is because of the way the tests are setup. Each test creates two widgets, one focusable and one not focusable (because it's disabled), triggers the `NextFocusAction`, then verifies that the focus is still on the focusable widget. For that to happen you need closed loop traversal. However, nothing in the test's widget structure favors closed loop traversal after my change. The tests, when running on the web, correctly lets the focus escape the `FlutterView`. That's the desired behavior. So I inject an artificial `FocusScope` to let the it test what it's trying to test.

Reply:
I see.  So if someone has an app where they depend on the focus looping without escape, they will need to add a `FocusScope`, but apps that want "normal" behavior will not need to add anything. There probably are some apps where this matters (a kiosk or something), but not many, and there's a workaround, so that's OK.